source/portal/socketIOServer.js

```diff
@@ -315,6 +332,16 @@ var Connection = function(spec, socket, reconnectionKey, portal, dock) {
     } else {
       pending_messages.push({event: event, data: data});
     }
+    let currentTime = process.hrtime()[0];
```

Comment:
why not var currentTime = process.hrtime();  and then use currentTime[0] + currentTime[1]/1000? Or you assume granity of second is enough?

Reply:
granity of second is enough for dropping old messages.